From b1efd619a751850c9ed972786c82b59121edd826 Mon Sep 17 00:00:00 2001
From: Gabriel Somlo <gsomlo@gmail.com>
Date: Wed, 15 Jul 2026 11:19:08 -0400
Subject: [PATCH 10401/15333] MAINTAINERS: remove Gabriel from LiteX and fw-cfg
 drivers

I no longer have the bandwidth to look after these drivers, so I'm
leaving them in the able hands of my co-maintainers.

Signed-off-by: Gabriel Somlo <gsomlo@gmail.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Message-ID: <20260715151908.1534002-1-gsomlo@gmail.com>
---
 MAINTAINERS | 2 --
 1 file changed, 2 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 5114e6db7307..3ea61eb118eb 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -15008,7 +15008,6 @@ F:	lib/tests/list-test.c
 LITEX PLATFORM
 M:	Karol Gugala <kgugala@antmicro.com>
 M:	Mateusz Holenko <mholenko@antmicro.com>
-M:	Gabriel Somlo <gsomlo@gmail.com>
 M:	Joel Stanley <joel@jms.id.au>
 S:	Maintained
 F:	Documentation/devicetree/bindings/*/litex,*.yaml
@@ -21919,7 +21918,6 @@ S:	Maintained
 F:	drivers/net/ipa/
 
 QEMU MACHINE EMULATOR AND VIRTUALIZER SUPPORT
-M:	Gabriel Somlo <somlo@cmu.edu>
 M:	"Michael S. Tsirkin" <mst@redhat.com>
 L:	qemu-devel@nongnu.org
 S:	Maintained
-- 
2.55.0

